tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -220,7 +265,7 @@ private static string GetManifestRegistry()
             JObject manifest = JObject.Parse(manifestJson);
             return (string)manifest["registry"];
         }
-
+  
```

Comment:
It looks like there are extra spaces in the blank line that were added.

Reply:
I can't see it, not sure where it is, can you please point out the line number?